doc/api/deprecations.md

```diff
@@ -2652,6 +2652,20 @@ Type: Documentation-only
 The [`crypto.Certificate()` constructor][] is deprecated. Use
 [static methods of `crypto.Certificate()`][] instead.
 
+### DEP0147: `fs.rmdir(path, { recursive: true })`
```

Comment:
I think it's supposed to be this?

```suggestion
### DEP0XXX: `fs.rmdir(path, { recursive: true })`
```

It gets assigned a number upon landing (although we seem to mess that up a fair amount--I certainly have more than once), just in case some other deprecation lands first.

On the other hand, maybe we stopped doing this or something? I don't see it documented anywhere. Someone who's been paying closer attention to this stuff, tell us what's up!

Reply:
I think `node-core-utils` will autoreplace `DEP0XXX` after https://github.com/nodejs/node-core-utils/pull/420. Which would mean the commit queue can as well.

Reply:
@richardlau @Trott if I'm understanding the process properly, now that #35579 has landed, with the documentation only deprecation, we should rebase this branch, and add the runtime deprecation as additional context to the deprecation that already exists?